ci/debian/centreon-ha-web.install

```diff
@@ -1,5 +1,6 @@
 etc/centreon-gorgone/config.d/cron.d/*      etc/centreon-gorgone/config.d/cron.d
 etc/centreon-ha/centreon_central_sync.pm    etc/centreon-ha
 etc/sudoers.d/centreon-cluster              etc/sudoers.d
-etc/sysconfig/*                             etc/sysconfig
+etc/default/*                               etc/default
 systemd/*                                   lib/systemd/system
+sed -i 's#etc/sysconfig#etc/defaul#g' systemd/*.service
```

Comment:
This doesn't work here, it will generate an error. Either you implement in `debian/rules` or put in `postinst`, like this:
```
sed -i 's#/etc/sysconfig#/etc/default#g' /lib/systemd/system/*.service
```